code/game/objects/items/storage/firstaid.dm

```diff
@@ -172,10 +172,10 @@
 	if(empty)
 		return
 	var/static/items_inside = list(
-		/obj/item/reagent_containers/syringe/antitoxin = 2,
+		/obj/item/reagent_containers/glass/bottle/antitoxin = 3,
```

Comment:
Stick with syringes or pills for medkits

Reply:
so... maybe just more syringes and less charcoal then?